### ISSUE TITLE
Quantization Noise model convert to torch.qint8

### ISSUE THREAD

**Issue by david-macleod** [pytorch/fairseq#2516]
Hi, I have trained a model using the steps in the Quantization Noise docs (Scalar Quantization) and was just wondering if there is a nice way to convert to a model which actually uses PyTorch torch.qint8 tensors/ops, rather than using fake quantization layers, e.g. equivalent to `torch.quantization.convert` used by PyTorch native quantization-aware training?

Also can we assume that the accuracy of a Quantization Noise model in eval model should give the same accuracy as one which has actually been converted to torch.qint8?

**Comment by stolam** [comment]
Hi, have you been able to resolve this?
